INFO - 2023-05-14 14:54:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 14:54:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 14:54:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:17:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:42:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:44:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:44:39 --> mysqli_sql_exception: Table 'posgrado.users' doesn't exist in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 E:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 E:\xampp\htdocs\codeigniter\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gma...', 'email', Array, NULL, 'email')
#6 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gma...', Array, Array)
#7 E:\xampp\htdocs\codeigniter\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(32): CodeIgniter\Controller->validate(Array)
#9 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#10 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#11 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-14 15:44:39 --> Table 'posgrado.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Persona.php(32): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:46:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:46:07 --> mysqli_sql_exception: Table 'posgrado.users' doesn't exist in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 E:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 E:\xampp\htdocs\codeigniter\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gma...', 'email', Array, NULL, 'email')
#6 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gma...', Array, Array)
#7 E:\xampp\htdocs\codeigniter\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(32): CodeIgniter\Controller->validate(Array)
#9 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#10 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#11 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-14 15:46:07 --> Table 'posgrado.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Persona.php(32): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:46:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:47:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:47:34 --> mysqli_sql_exception: Table 'posgrado.users' doesn't exist in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 E:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 E:\xampp\htdocs\codeigniter\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gma...', 'email', Array, NULL, 'email')
#6 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gma...', Array, Array)
#7 E:\xampp\htdocs\codeigniter\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(32): CodeIgniter\Controller->validate(Array)
#9 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#10 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#11 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-14 15:47:34 --> Table 'posgrado.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Persona.php(32): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:49:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:49:03 --> mysqli_sql_exception: Table 'posgrado.email' doesn't exist in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 E:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 E:\xampp\htdocs\codeigniter\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gma...', 'email', Array, NULL, 'email')
#6 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gma...', Array, Array)
#7 E:\xampp\htdocs\codeigniter\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(32): CodeIgniter\Controller->validate(Array)
#9 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#10 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#11 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-14 15:49:03 --> Table 'posgrado.email' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1
FROM `email`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `email`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `email`
WHERE `email` = \'lguevara240@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `email`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('lguevara240@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'lguevara240@gmail.com', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Persona.php(32): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:49:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:49:21 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(44): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:49:21 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(44): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:50:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:50:12 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(44): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:50:12 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(44): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:50:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:50:56 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(44): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:50:56 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(44): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:54:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:54:26 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(44): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:54:26 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(44): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:56:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:56:51 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(43): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:56:51 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(43): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:57:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:57:26 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(43): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:57:26 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(43): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:57:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:57:29 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(43): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:57:29 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(43): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:57:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:57:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:57:34 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(43): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:57:34 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(43): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:57:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:57:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:57:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:57:45 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(43): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:57:45 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(43): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 15:58:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:58:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 15:58:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 15:58:16 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(43): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 15:58:16 --> Column count doesn't match value count at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \', NOW());')
 4 APPPATH\Models\PersonaModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(43): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 16:00:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-05-14 16:00:22 --> strtoupper() expects exactly 1 argument, 4 given
in APPPATH\Models\PersonaModel.php on line 58.
 1 APPPATH\Models\PersonaModel.php(58): strtoupper('Leonardo Guevara ', 'lguevara240@gmail.com', '77346272', '9845916')
 2 APPPATH\Controllers\Persona.php(46): App\Models\PersonaModel->insertar([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 16:03:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-05-14 16:03:37 --> No se puede encontrar la ruta persona mientras se realiza el enrutamiento inverso.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 60.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(60): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('persona')
 2 APPPATH\Controllers\Persona.php(47): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 16:04:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:04:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:11:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:11:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:12:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 16:12:02 --> mysqli_sql_exception: Duplicate entry 'LGUEVARA240@GMAIL.COM' for key 'email' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(62): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(46): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 16:12:02 --> Duplicate entry 'LGUEVARA240@GMAIL.COM' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());')
 4 APPPATH\Models\PersonaModel.php(62): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?,?,?,?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(46): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 16:14:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 16:14:06 --> mysqli_sql_exception: Duplicate entry 'LGUEVARA240@GMAIL.COM' for key 'email' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(62): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(46): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 16:14:06 --> Duplicate entry 'LGUEVARA240@GMAIL.COM' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());')
 4 APPPATH\Models\PersonaModel.php(62): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?,?,?,?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(46): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 16:14:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:14:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 16:14:11 --> mysqli_sql_exception: Duplicate entry 'LGUEVARA240@GMAIL.COM' for key 'email' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pe...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 E:\xampp\htdocs\codeigniter\app\Models\PersonaModel.php(62): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(46): App\Models\PersonaModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 16:14:11 --> Duplicate entry 'LGUEVARA240@GMAIL.COM' for key 'email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( \'LEONARDO GUEVARA \',\'LGUEVARA240@GMAIL.COM\',\'77346272\',\'9845916\', NOW());')
 4 APPPATH\Models\PersonaModel.php(62): CodeIgniter\Database\BaseConnection->query('INSERT INTO `persona` (`nombre`, `email`, `telefono`, `cedula`, `created_at`) VALUES ( ?,?,?,?, NOW());', [...])
 5 APPPATH\Controllers\Persona.php(46): App\Models\PersonaModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 16:15:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:16:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:20:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:20:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:20:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:21:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:23:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:23:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:24:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:25:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:25:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:25:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:26:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 16:26:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:16:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 17:16:13 --> mysqli_sql_exception: Unknown column 'id' in 'where clause' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 E:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 E:\xampp\htdocs\codeigniter\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('LGUEVARA240@GMA...', 'email', Array, NULL, 'email')
#6 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'LGUEVARA240@GMA...', Array, Array)
#7 E:\xampp\htdocs\codeigniter\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(60): CodeIgniter\Controller->validate(Array)
#9 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->update()
#10 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#11 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-14 17:16:13 --> Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `id` != \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `id` != \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `id` != \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `persona`
WHERE `email` = :email:
AND `id` != :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('LGUEVARA240@GMAIL.COM', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'LGUEVARA240@GMAIL.COM', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Persona.php(60): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->update()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 17:16:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:16:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 17:16:50 --> mysqli_sql_exception: Unknown column 'id' in 'where clause' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 E:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 E:\xampp\htdocs\codeigniter\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('LGUEVARA240@GMA...', 'email', Array, NULL, 'email')
#6 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'LGUEVARA240@GMA...', Array, Array)
#7 E:\xampp\htdocs\codeigniter\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(60): CodeIgniter\Controller->validate(Array)
#9 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->update()
#10 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#11 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-14 17:16:50 --> Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `id` != \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `id` != \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `id` != \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `persona`
WHERE `email` = :email:
AND `id` != :id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('LGUEVARA240@GMAIL.COM', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'LGUEVARA240@GMAIL.COM', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Persona.php(60): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->update()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 17:17:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 17:17:09 --> mysqli_sql_exception: Unknown column 'iidpersonad' in 'where clause' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1\nFROM `...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 E:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 E:\xampp\htdocs\codeigniter\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('LGUEVARA240@GMA...', 'email', Array, NULL, 'email')
#6 E:\xampp\htdocs\codeigniter\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'LGUEVARA240@GMA...', Array, Array)
#7 E:\xampp\htdocs\codeigniter\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 E:\xampp\htdocs\codeigniter\app\Controllers\Persona.php(60): CodeIgniter\Controller->validate(Array)
#9 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Persona->update()
#10 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
#11 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-14 17:17:09 --> Unknown column 'iidpersonad' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `iidpersonad` != \'1\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `iidpersonad` != \'1\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `persona`
WHERE `email` = \'LGUEVARA240@GMAIL.COM\'
AND `iidpersonad` != \'1\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `persona`
WHERE `email` = :email:
AND `iidpersonad` != :iidpersonad:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(312): CodeIgniter\Validation\StrictRules\Rules->is_unique('LGUEVARA240@GMAIL.COM', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('email', 'email', 'LGUEVARA240@GMAIL.COM', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Persona.php(60): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Persona->update()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Persona))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 17:17:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:17:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:18:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:18:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:18:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:20:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:22:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:22:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:22:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:22:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:22:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:28:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:28:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:28:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:28:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:29:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:29:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:29:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:30:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:32:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:32:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:32:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:32:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:32:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:32:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:33:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:33:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:33:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:33:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:43:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:43:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:43:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:43:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:44:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:44:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:44:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:51:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:52:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:52:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:52:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:52:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:52:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 17:52:17 --> mysqli_sql_exception: Duplicate entry 'BOTELLA' for key 'name' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `un...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `un...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `un...')
#3 E:\xampp\htdocs\codeigniter\app\Models\UnidadModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `un...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Unidad.php(40): App\Models\UnidadModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Unidad->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidad))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 17:52:17 --> Duplicate entry 'BOTELLA' for key 'name'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `unidad` (`name`, `created_at`) VALUES ( \'BOTELLA\', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `unidad` (`name`, `created_at`) VALUES ( \'BOTELLA\', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `unidad` (`name`, `created_at`) VALUES ( \'BOTELLA\', NOW());')
 4 APPPATH\Models\UnidadModel.php(59): CodeIgniter\Database\BaseConnection->query('INSERT INTO `unidad` (`name`, `created_at`) VALUES ( ?, NOW());', [...])
 5 APPPATH\Controllers\Unidad.php(40): App\Models\UnidadModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Unidad->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidad))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 17:53:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:55:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:55:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:55:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:55:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:56:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 17:56:15 --> mysqli_sql_exception: Unknown column 'password' in 'field list' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 E:\xampp\htdocs\codeigniter\app\Models\UsuarioModel.php(83): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Usuario.php(91): App\Models\UsuarioModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Usuario->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 17:56:15 --> Unknown column 'password' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( \'1\',\'ae24a083cd1708ffbdaa6e21eb66010b6af747b1360d150b315546980a3f7328d6c636f266251f02287947a5d604adc3636282c990ec9ca0bc1a508fab3506ce\',\'lkkl\',\'fibo\',\'hola\',\'1\', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( \'1\',\'ae24a083cd1708ffbdaa6e21eb66010b6af747b1360d150b315546980a3f7328d6c636f266251f02287947a5d604adc3636282c990ec9ca0bc1a508fab3506ce\',\'lkkl\',\'fibo\',\'hola\',\'1\', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( \'1\',\'ae24a083cd1708ffbdaa6e21eb66010b6af747b1360d150b315546980a3f7328d6c636f266251f02287947a5d604adc3636282c990ec9ca0bc1a508fab3506ce\',\'lkkl\',\'fibo\',\'hola\',\'1\', NOW());')
 4 APPPATH\Models\UsuarioModel.php(83): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( ?,?,?,?,?,?, NOW());', [...])
 5 APPPATH\Controllers\Usuario.php(91): App\Models\UsuarioModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuario->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-05-14 17:56:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-14 17:57:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-14 17:57:02 --> mysqli_sql_exception: Unknown column 'password' in 'field list' in E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 E:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 E:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 E:\xampp\htdocs\codeigniter\app\Models\UsuarioModel.php(83): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array)
#4 E:\xampp\htdocs\codeigniter\app\Controllers\Usuario.php(91): App\Models\UsuarioModel->insertar(Array)
#5 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(934): App\Controllers\Usuario->insert()
#6 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#7 E:\xampp\htdocs\codeigniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp\htdocs\codeigniter\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-05-14 17:57:02 --> Unknown column 'password' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( \'1\',\'2fe2b19cbb0c918942b84acbc4bba5ebaac54a144d7498cbd4fa10126594493b5d66b3221660cdad1f92c7d4972c91ff961c661a0e241ccb705d7bb2bb232326\',\'lkkl\',\'fibo\',\'hol\',\'1\', NOW());', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( \'1\',\'2fe2b19cbb0c918942b84acbc4bba5ebaac54a144d7498cbd4fa10126594493b5d66b3221660cdad1f92c7d4972c91ff961c661a0e241ccb705d7bb2bb232326\',\'lkkl\',\'fibo\',\'hol\',\'1\', NOW());')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( \'1\',\'2fe2b19cbb0c918942b84acbc4bba5ebaac54a144d7498cbd4fa10126594493b5d66b3221660cdad1f92c7d4972c91ff961c661a0e241ccb705d7bb2bb232326\',\'lkkl\',\'fibo\',\'hol\',\'1\', NOW());')
 4 APPPATH\Models\UsuarioModel.php(83): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`fk_persona`, `password`, `foto`, `user`, `detalle`, `fkroles`, `created_at`) 
        VALUES ( ?,?,?,?,?,?, NOW());', [...])
 5 APPPATH\Controllers\Usuario.php(91): App\Models\UsuarioModel->insertar([...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuario->insert()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
